docs/user-guide/examples.rst

```diff
@@ -0,0 +1,68 @@
+########
+Examples
+########
+
+The `SDM Schemas github repository <https://github.com/lsst/sdm_schemas>`_ contains Felis schema files used by the Rubin Observatory, which will be used as examples to illustrate the features of Felis.
+The following is an excerpt from the `DP0.2 DC2 schema <https://github.com/lsst/sdm_schemas/blob/main/yml/dp02_dc2.yaml>`_:
+
+.. code-block:: yaml
+   :linenos:
+
+   ---
+   name: dp02_dc2_catalogs
+   "@id": "#dp02_dc2_catalogs"
+   description: "Data Preview 0.2 contains the image and catalog products of the Rubin Science
+   Pipelines v23 processing of the DESC Data Challenge 2 simulation, which covered 300 square
+   degrees of the wide-fast-deep LSST survey region over 5 years."
+   tables:
+   - name: Object
+     "@id": "#Object"
+      description: "Properties of the astronomical objects detected and measured on the deep coadded images."
+      tap:table_index: 1
+      columns:
+      - name: objectId
+        "@id": "#Object.objectId"
+        datatype: long
+        description: Unique id. Unique ObjectID
+        ivoa:ucd: meta.id;src;meta.main
+      - name: coord_ra
+        "@id": "#Object.coord_ra"
+        datatype: double
+        description: Fiducial ICRS Right Ascension of centroid used for database indexing
+        fits:tunit: deg
+        ivoa:ucd: pos.eq.ra;meta.main
+
+Lines 2-6 define the schema name, id, and description.
+Name and id are always required.
```

Comment:
Could you explain the use of `#` in `@id`?

Reply:
I can't really explain it other than "Someone did it initially and everyone else copied it."

I guess it means "in this file" like an HTML link? Then if you were combining names the references could look like "myschema#myid."

I'm not entirely sure though. There's no requirement that the IDs have "#" in them. It is just a convention.